qdrant-landing/content/documentation/tutorials/async-api.md

```diff
@@ -0,0 +1,362 @@
+    optimizer_config {
+      deleted_threshold: 0.2
+      vacuum_min_vector_number: 1000
+      default_segment_number: 0
+      indexing_threshold: 20000
+      flush_interval_sec: 5
+      max_optimization_threads: 1
+    }
+    wal_config {
+      wal_capacity_mb: 32
+      wal_segments_ahead: 0
+    }
+    quantization_config {
+      scalar {
+        type: Int8
+        always_ram: false
+      }
+    }
+  }
+  indexed_vectors_count: 0
+}
+time: 1.3022e-05
+```
+
+As you can see, the collection was created properly and is ready to be used.
+
+##### Handling exceptions
+
+If the collection does not exist, the server will return an error. We can handle it by catching the exception. We may expect
+some specific exceptions, or simply catch all the errors that subclass `RpcError` from `grpc` package:
+
+```python
+from grpc import RpcError
+
+try:
+    response = await client.async_grpc_collections.Get(
+        grpc.GetCollectionInfoRequest(
+            collection_name="my_collection_2"
+        )
+    )
+except RpcError as e:
+    print(e)
+```
+
+Our exception contain the details about the error, including the status code and the message:
+
+```text
+<AioRpcError of RPC that terminated with:
+	status = StatusCode.NOT_FOUND
+	details = "Not found: Collection `my_collection_2` doesn't exist!"
+	debug_error_string = "UNKNOWN:Error received from peer  {created_time:"2023-08-29T12:14:05.885634788+02:00", grpc_status:5, grpc_message:"Not found: Collection `my_collection_2` doesn\'t exist!"}"
+>
+```
+
+We can use those details to properly handle some specific cases. For example, if we want to create a collection only if it does not
+exist, we can check the status code and create it if it is `StatusCode.NOT_FOUND`:
+
+#### Updating collection configuration
+
+It's quite likely that we will want to update the collection configuration. For example, we may want to change the HNSW parameters to improve
+the search precision. Let's change the `ef_construct` parameter to 200:
+
+```python
+await client.async_grpc_collections.Update(
+    grpc.UpdateCollection(
+        collection_name="my_collection",
+        hnsw_config=grpc.HnswConfigDiff(
+            ef_construct=200,
+        ),
+    )
+)
+```
+
+#### Deleting a collection
+
+There is plenty of other methods available for collections, like creating aliases or checking the status of the cluster. We are not going
+to call all of them in this tutorial, but you can always [check the list of available methods and their parameters in the 
+documentation](https://github.com/qdrant/qdrant/blob/master/docs/grpc/docs.md#collections_serviceproto).
+
+Right now, we are just going to delete the collection we created:
+
+```python
+await client.async_grpc_collections.Delete(
+    grpc.DeleteCollection(
+        collection_name="my_collection"
+    )
+)
+```
+
+That's it when it comes to collection management. Let's move on to points, as semantic search requires us to have some vectors in the
+collection. 
```

Comment:
```suggestion
That's it when it comes to collection management. Let's move on to points (individual vectors) allowing us to load data into our collection
```

I wanted to say "points (observations)" but was not sure if that would be understood by the audience.